### DISCUSSION TITLE
youtube-deno : A Deno client for the YouTube Data API

### DISCUSSION THREAD

**Discussion by akshgpt7** [nestdotland/nest.land#222] [Show and tell]
https://github.com/akshgpt7/youtube-deno

Lets you incorporate core YouTube features, such as uploading videos, creating and managing playlists, searching for content, and almost any interaction with YouTube in your Deno application.

Any feedback or contributions are much welcome!
Hope this is helpful for someone.

**Comment by t8** [comment]
This is a really neat project! `youtube-deno` is the first Deno module I've seen that interfaces with the YouTube API. I really like how you organized the functions the names appear to be very simple and easy to understand.

Great work, and thanks for sharing here!
